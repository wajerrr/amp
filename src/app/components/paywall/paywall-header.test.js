import React from 'react';
import renderer from 'react-test-renderer';
import PaywallHeader from './paywall-header';

describe('PaywallHeader', () => {
  it('should match the snapshot', () => {
    const paywallHeader = renderer
      .create(
        <PaywallHeader
          data={{
            id: 'regwall-header',
            type: 'header',
            content: {
              title: 'Title',
              text: ['Test header'],
            },
          }}
        />
      )
      .toJSON();
    expect(paywallHeader).toMatchSnapshot();
  });
});
