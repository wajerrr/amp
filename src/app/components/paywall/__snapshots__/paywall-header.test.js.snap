// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaywallHeader should match the snapshot 1`] = `
<header
  className={undefined}
>
  <h2
    className={undefined}
  >
    Title
  </h2>
  <p
    className={undefined}
  >
    Test header
  </p>
</header>
`;
